x-pack/plugins/security_solution/public/entity_analytics/components/risk_score_enable_section.tsx

```diff
@@ -345,14 +331,19 @@ export const RiskScoreEnableSection = () => {
         </EuiTitle>
         <EuiSpacer />
         <ul>
-          {docsLinks.map(({ link, label }) => (
-            <li key={link}>
-              <EuiLink href={link} target="_blank" external>
-                {label}
-              </EuiLink>
-              <EuiSpacer size="s" />
-            </li>
-          ))}
+          <li key={DETECTION_ENTITY_DASHBOARD}>
+            <EuiLink href={DETECTION_ENTITY_DASHBOARD} target="_blank" external>
```

Comment:
Per @SourinPaul , based on [this comment](https://github.com/elastic/security-team/issues/7920#issuecomment-1795835137), this link should actually take you to the in-product Entity Analytics Dashboard page instead of the docs page.

Reply:
Your understanding is correct, @jaredburgettelastic

Reply:
Done